Python/PDF_Reader/README.md

```diff
@@ -0,0 +1,24 @@
+# PDF_Reader
+
+At times, we feel like having an assistant which could read our lengthy documents for us and we could keep listening to it. This script is very useful in such cases. Whenever the user specifies the path to the file in their system, and execute this program, the pdf is read aloud.

```

Comment:
Can you give a general understanding: 

`This Script allows us to specify the path to a PDF File and when given an input in form of the page number, reads it aloud for the user....` 

Use the standard documentation principles to help people have the understanding what the script is supposed to do.